Python/Product/Analyzer/Intellisense/OutOfProcProjectAnalyzer.cs

```diff
@@ -281,10 +281,14 @@ sealed class OutOfProcProjectAnalyzer : IDisposable {
 
             var interpreter = factory.CreateInterpreter();
             if (interpreter != null) {
-                _pyAnalyzer = PythonAnalyzer.Create(factory, interpreter);
-                ReloadTask = _pyAnalyzer.ReloadModulesAsync()/*.HandleAllExceptions(_serviceProvider, GetType())*/;
-                ReloadTask.ContinueWith(_ => ReloadTask = null);
-                interpreter.ModuleNamesChanged += OnModulesChanged;
+                try {
+                    _pyAnalyzer = PythonAnalyzer.Create(factory, interpreter);
+                    ReloadTask = _pyAnalyzer.ReloadModulesAsync()/*.HandleAllExceptions(_serviceProvider, GetType())*/;
+                    ReloadTask.ContinueWith(_ => ReloadTask = null);
+                    interpreter.ModuleNamesChanged += OnModulesChanged;
+                } catch (InvalidOperationException ex) {
+                    error = ex.ToString();
```

Comment:
Is that meant to be removed (you don't do anything with `error` after this block)?